client/cmd/dexc/ui/accountsview.go

```diff
@@ -0,0 +1,135 @@
+
+package ui
+
+import (
+	"github.com/rivo/tview"
+)
+
+var dummyFormBox = tview.NewBox()
+
+// accountsViewer is a view for manipulating DEX account and wallet settings.
+type accountsViewer struct {
+	*tview.Flex
+	chain *focusChain
+	form  *tview.Grid
+}
+
+// newAccountsView is the constructor for an accountsViewer.
+func newAccountsView() *accountsViewer {
+	// A journal for logging account-related messages.
+	acctsJournal := newJournal("Accounts Journal", nil)
+	// acctsLog is global.
+	acctsLog = NewLogger("ACCTS", acctsJournal.Write)
+	formBox := tview.NewGrid()
+	formBox.SetBackgroundColor(colorBlack)
+	var acctForm *tview.Form
+
+	// The list of available DEX accounts, and a button to display a form to add
+	// a new DEX account.
+	acctsList := newChooser("Accounts", nil)
+	newAcctBttn := newSimpleButton("Add New DEX Account", func() {
+		formBox.Clear()
+		formBox.AddItem(acctForm, 0, 0, 1, 1, 0, 0, false)
+		acctForm.SetBorderColor(focusColor)
+		app.SetFocus(acctForm)
+	})
+	acctsColumn := tview.NewFlex().
+		SetDirection(tview.FlexRow).
+		AddItem(acctsList, 0, 1, false).
+		AddItem(newAcctBttn, 3, 0, false)
+
+	// The list of exchange wallets registered, and a button to dipslay a form
+	// to add a new wallet.
+	walletsList := newChooser("Wallets", nil)
+	newWalletBttn := newSimpleButton("Add Wallet", func() {
+		acctsLog.Errorf("cannot add a wallet without an account")
+	})
+	walletsColumn := tview.NewFlex().
+		SetDirection(tview.FlexRow).
+		AddItem(walletsList, 0, 1, false).
+		AddItem(newWalletBttn, 3, 0, false)
+
+	// The third column is the main content area (formBox), and a log display at
+	// the bottom
+	thirdColumn := tview.NewFlex().
+		SetDirection(tview.FlexRow).
+		AddItem(fullyCentered(formBox, 60, 17), 0, 3, false).
+		AddItem(acctsJournal, 0, 1, false)
+
+	// The main view.
+	wgt := tview.NewFlex().
+		AddItem(acctsColumn, 30, 0, false).
+		AddItem(walletsColumn, 30, 0, false).
+		AddItem(thirdColumn, 0, 1, false)
+
+	// A form to add a DEX account.
+	//
+	// DRAFT NOTE: I think we need more control over forms and should implement
+	// them using tview.Grid, but this is a start. A major downside is that
+	// tview.Form hijacks tab and arrow keys, making for some unintuitive
+	// navigation behavior.
+	var acctURL, dcrAcctName, dcrwAddr, dcrwPW, dcrwRPCUser, dcrwRPCPass string
+	acctForm = tview.NewForm().
+		AddInputField("DEX URL", "", 0, nil, func(url string) {
+			acctURL = url
+		}).
+		AddInputField("Account Name", "", 0, nil, func(name string) {
+			dcrAcctName = name
+		}).
+		AddInputField("dcrwallet RPC Address", "", 0, nil, func(name string) {
+			dcrwAddr = name
+		}).
+		AddInputField("RPC Username", "", 0, nil, func(name string) {
+			dcrwRPCUser = name
+		}).
+		AddPasswordField("RPC Password", "", 0, 0, func(name string) {
+			dcrwRPCPass = name
+		}).
+		AddPasswordField("Wallet Password", "", 0, 0, func(pw string) {
+			dcrwPW = pw
+		}).
+		AddButton("register", func() {
```

Comment:
Once I get focus to the register button, I can't seem to get back to the text fields.

Reply:
NVM, I figured out it was TAB.